### PR TITLE
telemetry: misc scripts

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: user.email <sage@telemetry.ceph.com>

### CODE REVIEW

File: src/telemetry/dump_from_es.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import json
+from server.ceph_telemetry.rest import Report
+
+f = open('es_dump.txt', 'r')
+a = f.read()
+
+j = json.loads(a)
+reports = j['hits']['hits']
+print len(reports)
```
Comment (sebastian-philipp) [inline src/telemetry/dump_from_es.py]:
```suggestion
print(len(reports))
```

File: src/telemetry/es_dump_to_file.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from os import path
+import json
+
+f = open('es_dump.txt', 'r')
+a = f.read()
+
+j = json.loads(a)
+reports = j['hits']['hits']
+print len(reports)
```
Comment (sebastian-philipp) [inline src/telemetry/es_dump_to_file.py]:
```suggestion
print(len(reports)(
```

File: src/telemetry/es_dump_to_file.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    fn = '/opt/telemetry/raw/' + rid + '.' + ts
+    
+    if path.exists(fn):
+        exists += 1
+    else:
+        f = open(fn, 'w')
+        f.write(json.dumps(r))
+        f.close()
+
+print 'total %d, bad %d, exists %d' % (total, bad, exists)
```
Comment (sebastian-philipp) [inline src/telemetry/es_dump_to_file.py]:
```suggestion
print('total %d, bad %d, exists %d' % (total, bad, exists))
```

File: src/telemetry/proc_reports.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    ccur = conn.cursor()
+
+    # cluster
+    ccur.execute("SELECT cluster_id,latest_report_stamp FROM cluster WHERE cluster_id=%s", (cluster_id,))
+    crows = ccur.fetchall()
+    update = True
+    if crows:
+        crow = crows[0]
+        if str(crow[1]) > ts:
+            print 'cluster %s already has newer report %s' % (cluster_id, crow[1])
```
Comment (sebastian-philipp) [inline src/telemetry/proc_reports.py]:
```suggestion
            print('cluster %s already has newer report %s' % (cluster_id, crow[1]))
```

File: src/telemetry/proc_reports.py [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+            update = False
+
+    if update:
+        print 'updating %s' % (cluster_id)
```
Comment (sebastian-philipp) [inline src/telemetry/proc_reports.py]:
```suggestion
        print('updating %s' % (cluster_id))
```

**Comment by liewegas** [comment]
@dmick added a commit to add another table to tables.txt. Let's commit this?  or you can push whatever you have cooking?

File: src/telemetry/tables.txt [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+       cluster_id VARCHAR(50),
+       raw_report TEXT,
+
+       timestamp TIMESTAMP,
+       entity_name CHAR(64),
+
+       version VARCHAR(64),
+
+       stack_sig VARCHAR(64),
+       stack_weak_sig VARCHAR(64),
```
Comment (dmick) [inline src/telemetry/tables.txt]: can remove this

File: src/telemetry/tables.txt [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+       stack TEXT
+);
+
+
+drop table cluster;
+drop table cluster_version;
+drop table crash;
```
Comment (dmick) [inline src/telemetry/tables.txt]: Why are these here?
